src/config/environment.js

```diff
@@ -1,5 +1,5 @@
 const config = {
-  backendUrl: 'http://localhost:3000/api/v1'
+  backendUrl: 'https://api.operationcode.org/api/v1'
```

Comment:
Does this mean all dev environments will be hitting the prod instance of our API?